watchfrr/watchfrr.c

```diff
@@ -1030,6 +1030,12 @@ void watchfrr_status(struct vty *vty)
 	struct timeval delay;
 
 	vty_out(vty, "watchfrr global phase: %s\n", phase_str[gs.phase]);
+	vty_out(vty, " Restart Command: %s\n", gs.restart_command);
```

Comment:
Since this is a shell command specified as an arg (might even contain a newline), it might be useful to use `%pSQq` here to escape the strings :)

Reply:
fixed